Dockerfile

```diff
@@ -1,6 +1,13 @@
-FROM python:3.12.4-slim AS production
+ARG PLATFORM=linux/x86_64
+ARG BASE_IMAGE=python:3.12.5-slim
```

Comment:
Just moving from `python:3.12.4-slim` to `python:3.12.5-slim` fixed the issue with running `apt-get` in FIPS-mode.  Apparently, `python:3.12.4-slim` was last built before the issue was fixed.